build/tenuki.js

```diff
@@ -550,7 +550,7 @@ var GameState = function GameState(_ref) {
 
 GameState.prototype = {
   _nextColor: function _nextColor() {
-    if (!this.color || this.color == "white") {
+    if (this.color == "white") {
```

Comment:
An extra argument in favor of this: the "current move" is arguably just the point that was played, not a full object representing state.